src/store/models/market/categorical.js

```diff
@@ -9,11 +9,13 @@ const CategoricalMarketRecord = Record({
   resolution: undefined, // moment
   volume: undefined, // decimal
   outcomes: undefined, // List<Outcome>
+  eventAddress: undefined, // string
   winningOutcome: undefined, // OutcomeRecord
   funding: undefined, // int
   creation: undefined, // moment
-  stage: undefined,
+  stage: undefined, // int
```

Comment:
Looking at all these comments, I think that it would be good to introduce flow 😄

Reply:
I agree, but let's do that at a later point :)